R/baggr_compare.R

```diff
@@ -1,5 +1,17 @@
 #' (Run and) compare multiple baggr models
 #'
+#' @description Compare multiple baggr models by either
+#' specifying multiple baggr models you have run or
+#' passing parameters to run a baggr model. If you pass
+#' existing models, you will get a layered effect plot
+#' of the treatment effects for the different models.
+#' If you pass parameters to the function you must specify
```

Comment:
Starting from "if you pass" could go to Details

Reply:
Did this in next update.